cdc/sinkv2/tablesink/event_table_sink.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tablesink
+
+import (
+	"sort"
+
+	"github.com/pingcap/tiflow/cdc/model"
+	"github.com/pingcap/tiflow/cdc/processor/pipeline"
+	"github.com/pingcap/tiflow/cdc/sinkv2/eventsink"
+	"github.com/pingcap/tiflow/cdc/sinkv2/tableevent"
+)
+
+// Assert TableSink implementation
+var _ TableSink = (*eventTableSink[*model.RowChangedEvent])(nil)
+var _ TableSink = (*eventTableSink[*model.SingleTableTxn])(nil)
+
+type eventTableSink[E tableevent.TableEvent] struct {
```

Comment:
Could we rename `eventTableSink` to `tableSinkImpl` and move related codes to `table_sink.go`?

Reply:
I think eventTableSink is more appropriate because it makes it clearer that it is a tablesink implementation of multiple events.